### ISSUE TITLE
Data likely corrupted on quit

### ISSUE THREAD

**Issue by dergoegge** [mit-dci/utreexo#234]
I ran into this on the test server.
```
On block : 426901
On block : 427001
On block : 427101
^CUser exit signal received. Exiting...
Program timed out. Force quitting. Data likely corrupted
```

https://github.com/mit-dci/utreexo/blob/1a03a98e9424108d4a22f15dd6f581b8b906009d/bridgenode/genproofs.go#L209-L215

I don't suspect that there is a bug but rather that the test server is too slow to finish up in 60sec. I was using the cow forest for which it is normal to take longer when quitting since it has to commit a bunch of data to disk.
We should remove the timeout or increase it.

**Comment by kcalvinalvin** [comment]
Yeah this has been on my TODO for a bit... if the user's disk is slow, it'll time out.

An easy fix would be to just increase it. Another (more elegant) fix could be to only force quit when given SIGINT twice.
